### ISSUE TITLE
[FEATURE REQUEST] Hide ignored packages

### ISSUE THREAD

**Issue by Cris--** [marticliment/UniGetUI#2609]
### Please confirm these before moving forward.

- [X] I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29).

### Describe the new feature

Add a filter "Hide ignored packages" for installed packages so the list is not cluttered with these ignored packages.

### Describe how this new feature could help users

It would really help me to overview the relevant ones with over 880 installed packages from all sources (mostly plugins for DAWs and graphic/video editing).

**Comment by slrgt** [comment]
i still get notifications that its trying to update ignored packages too

**Comment by Joshua66252** [comment]
I would also greatly appreciate this. (Mostly for WinGet)

WinGet loves to manage everything including packages it didn't install, ex, I manage Kdenlive with Chocolate, there are two listings on my installed list, the duplicate being WinGet. Quite a few packages have this issue, it can make the installed list feel cluttered. While you can toggle WinGet as a source in the installed list, I have packages I manage with WinGet that would be missing.
For better usability a separate feature, perhaps called "blacklist" or "hide" that works on a per-package basis would be nice, hiding all packages with ignored updates could be annoying if specific outdated versions of packages are for any reason needed and have to be ignored.

TL;DR
Adding a per-package blacklist/hide option would help reduce clutter in the installed list and be a nice quality of life feature.
